### ISSUE TITLE
K-Type: "Failed to load layout" error

### ISSUE THREAD

**Issue by bennettp123** [kiibohd/configurator#146]
Repro steps:

1. Install the macOS build, version 1.0.1
2. Launch Kiibohd Configurator
3. Click "K-Type"

Expected result: 
* Configuration options for the K-Type are shown

Actual result:
* At step 3, an error appears: "Failed to load layout"
* Configuration options for the K-Type are not shown
* The following errors are logged in the DevTools console: 

```
index.js:54 GET http://vash.input.club:3001/layouts/KType-Standard.json net::ERR_CONNECTION_REFUSED
```

```
app.js:43 Uncaught (in promise) TypeError: Failed to fetch
```

```
vash.input.club:3001/versions:1 Failed to load resource: net::ERR_CONNECTION_REFUSED
```

**Comment by bennettp123** [comment]
After digging a little deeper, this doesn't look like a genuine outage, just port 3001 blocked on my corporate network. 🥇

However, I'd be happy to contribute some of my time to migrate this to port 443, if there's a desire to do so. :)

**Comment by bennettp123** [comment]
Looks like this is fixed in 1.0.2 🎉 

Relevant commit: https://github.com/kiibohd/configurator/commit/18ac1cd950fb16c0985da50340d2560a0b525022
